### PR TITLE
Refresh of the other/RKE2 installation.

### PR DESCRIPTION
**Pull request by jhkrug**
I have added a demo of what running the command might look like. Using vhs from https://github.com/charmbracelet/vhs. Like ascinema but different. The playback is specified in rke2.tape and then processed to produce the output gif.

The gif is wrapped up in a details sections so its not being annoying while your are trying to focus. An advantage is that it only starts playing when the details is expanded. Helps avoid being lost in the middle.

What do you think? More of this?

### CODE REVIEW

File: docs/installation/other_inst_scenarios/install_epinio_on_rke.md [hunk 1]
```diff
@@ -1,76 +1,111 @@
...
+title: "Installing Epinio on RKE2"
+description: Installing Epinio with RKE2. The needed environment and prerequisites.
+keywords: [epinio, k8s, kubernetes, rke2, installation]
 ---
 
-# Installing Epinio On RKE2
-
-This guide will help you to deploy a suitable RKE2 Kubernetes cluster for Epinio. More details can be found in RKE2 [quickstart](https://docs.rke2.io/install/quickstart/) guide.
+This guide will help you to deploy a suitable RKE2 Kubernetes cluster for Epinio.
+More details for installing RKE2, can be found in the RKE2 [quickstart](https://docs.rke2.io/install/quickstart/) guide.
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rke.md:10]:
```suggestion
More details for installing RKE2 can be found in the RKE2 [quickstart](https://docs.rke2.io/install/quickstart/) guide.
```
comma or no comma ? to me it reads better without.

File: docs/installation/other_inst_scenarios/install_epinio_on_rke.md [hunk 2]
```diff
@@ -1,76 +1,111 @@
...
+    To support Epinio a storage provisioner is needed.
+    You can use any storage provisioner which provides, `ReadWriteMany` (RWX) Access Mode and a **default StorageClass** resource for dynamic storage provisioning.
+
+    :::note
+
+    To verify that your cluster provides a default StorageClass run the command  `kubectl get storageclass`. The default StorageClass is marked with the string `(default)` next to its name in the output list.
+
+    :::
+
+    As an example, you can deploy and configure `local-path` dynamic storage provisioner by running:
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rke.md:73]:
```suggestion
    As an example, you can deploy and configure a `local-path` dynamic storage provisioner by running:
```

File: docs/installation/other_inst_scenarios/rke2.tape [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Sleep 1s
+
+Type "curl -sfL https://get.rke2.io | sh -"
+Sleep 500ms
+Enter
+Sleep 10s
+Type "systemctl enable --now rke2-server.service"
+Sleep 500ms
+Enter
+Sleep 50s
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/rke2.tape:30]: This is a lot of time waiting for anything, with no visible response at all.
Can we `Type "# ... skipping waiting for a minute"` here and reduce the sleep to `500ms` ?
Problem with that is that this file looks to be an instruction stream driving a terminal.
The command responses are not shown, and will appear only in the result, I suspect.

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Some small changes. Not fully sure about them.

Regarding the gif/asciinema ....

On one hand it feels a bit like a gimmick. All the necessary commands are listed in the text, and can be copy/pasted from there into the local terminal for execution.

On the other hand this does provide a demonstration/proof that the commands did work at the time of writing, and also what the responses from the commands should look like in general. Putting that in the text would heavily interrupt the flow of reading.

In the balance ... Yes, keep it.

File: docs/installation/other_inst_scenarios/rke2.tape (hunk 3 above)
Comment (jhkrug) [reply docs/installation/other_inst_scenarios/rke2.tape:30]: I'm sure something better is possible. It's just the (my) first iteration with it.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**